cpp/ql/test/library-tests/ir/ir/aliased_ir.expected

```diff
@@ -18177,6 +18177,64 @@ ir.cpp:
+# 2536|     m2536_3(int)                        = Store[#throw2536:5]                  : &:r2536_1, r2536_2
+# 2536|     v2536_4(void)                       = ThrowValue                           : &:r2536_1, m2536_3
+#-----|   Exception -> Block 3
+
+# 2533|   Block 1
+# 2533|     m2533_5(unknown) = Phi          : from 2:~m2534_6, from 4:~m2540_14
+# 2533|     v2533_6(void)    = AliasedUse   : ~m2533_5
+# 2533|     v2533_7(void)    = ExitFunction : 
+
+# 2533|   Block 2
+# 2533|     v2533_8(void) = Unwind : 
+#-----|   Goto -> Block 1
+
+# 2538|   Block 3
+# 2538|     v2538_1(void) = CatchByType[char] : 
+#-----|   Exception -> Block 2
+#-----|   Goto -> Block 4
+
+# 2538|   Block 4
+# 2538|     r2538_2(glval<char>)                 = VariableAddress[(unnamed parameter 0)]     : 
+# 2538|     m2538_3(char)                        = InitializeParameter[(unnamed parameter 0)] : &:r2538_2
+# 2538|     v2538_4(void)                        = NoOp                                       : 
+# 2540|     r2540_1(glval<ClassWithDestructor>)  = VariableAddress[x]                         : 
+# 2540|     r2540_2(glval<unknown>)              = FunctionAddress[~ClassWithDestructor]      : 
+# 2540|     v2540_3(void)                        = Call[~ClassWithDestructor]                 : func:r2540_2, this:r2540_1
+# 2540|     m2540_4(unknown)                     = ^CallSideEffect                            : ~m2534_6
+# 2540|     m2540_5(unknown)                     = Chi                                        : total:m2534_6, partial:m2540_4
+# 2540|     v2540_6(void)                        = ^IndirectReadSideEffect[-1]                : &:r2540_1, m2534_8
+# 2540|     m2540_7(ClassWithDestructor)         = ^IndirectMayWriteSideEffect[-1]            : &:r2540_1
+# 2540|     m2540_8(ClassWithDestructor)         = Chi                                        : total:m2534_8, partial:m2540_7
+# 2540|     v2540_9(void)                        = NoOp                                       : 
+# 2540|     r2540_10(glval<ClassWithDestructor>) = VariableAddress[x]                         : 
+# 2540|     r2540_11(glval<unknown>)             = FunctionAddress[~ClassWithDestructor]      : 
+# 2540|     v2540_12(void)                       = Call[~ClassWithDestructor]                 : func:r2540_11, this:r2540_10
+# 2540|     m2540_13(unknown)                    = ^CallSideEffect                            : ~m2540_5
+# 2540|     m2540_14(unknown)                    = Chi                                        : total:m2540_5, partial:m2540_13
+# 2540|     v2540_15(void)                       = ^IndirectReadSideEffect[-1]                : &:r2540_10, m2540_8
+# 2540|     m2540_16(ClassWithDestructor)        = ^IndirectMayWriteSideEffect[-1]            : &:r2540_10
+# 2540|     m2540_17(ClassWithDestructor)        = Chi                                        : total:m2540_8, partial:m2540_16
```

Comment:
The destructor for `x` is called twice here. This is incorrect, as one is associated with the return from the function, while the other is associated with further unwinding of the stack.